suzieq/sqobjects/basicobj.py

```diff
@@ -223,7 +224,7 @@ def unique(self, **kwargs) -> pd.DataFrame:
         columns = kwargs.pop('columns', self.columns)
 
         if columns is None or columns == ['default']:
-            raise ValueError('Must specify columns with unique')
+            columns = self.default_columns
```

Comment:
User can specify '*' which is all columns instead of nothing or 'default', and we need to reject that as well. The original code had this bug too, just noticing it.

Reply:
Ok, I'll add that check too.